src/me.rs

```diff
@@ -138,6 +138,54 @@ pub fn motion_estimation(
   }
 }
 
+pub fn estimate_motion_ss4(
```

Comment:
ss means 's'ub's'ampled? :) I am always curious about the shortened names in computer programs!